src/data/linux-quiz.ts

```diff
@@ -1655,6 +1655,16 @@ const linuxQuiz = [
     Explanation:
       'You can use the usermod command to add a user to a group. The -a parameter means "append" and the -G parameter adds a group as a secondary group',
     Link: "https://www.freecodecamp.org/news/linux-user-groups-explained-how-to-add-a-new-group-a-new-group-member-and-change-groups/"
+  },
+  {
+    Question:
+      "Which command below doesn't display logs?",
+    Answer: "cat /etc/logcheck/ignore.d.server/rsyslog",
```

Comment:
let's pick a simpler answer

```suggestion
    Answer: "cd",
```